### ISSUE TITLE
Investigate and improve documentation of this repos commands

### ISSUE THREAD

**Issue by wintondeshong** [AndcultureCode/AndcultureCode.Cli#59]
The current COMMANDS.md is fine, but at this current size has become a bit hard to read and manage. Investigate options and port existing documentation.

**Comment by brandongregoryscott** [comment]
I wonder if this would be as simple as pulling in [Typedoc](https://typedoc.org/) once issue #14 is complete? It could be a good starter, at least, for a simple way to automate some of our documentation until we further refine it.

**Comment by wintondeshong** [comment]
Yeah, I agree. Once we have the typescript port we could quickly follow it up with typedoc
